gsms.py

```diff
@@ -53,15 +53,83 @@
 
 # standard imports
 import argparse
+import ctypes
+from ctypes import wintypes
 import json
 import os
+import re
 import shutil
 import time
+import uuid
 
 # lib imports
 import pylnk3
 
 
+# Code from here and modified to work in this project
+# https://gist.github.com/mkropat/7550097
+class GUID(ctypes.Structure):
+    """
+    Class to build a GUID compliant object for use in WIndows libraries
+
+    Parameters
+    -------
+    uuid: string
+
+    Returns
+    -------
+    GUID object
+
+    Examples
+    --------
+    >>> GUID(uuid.UUID("{B4BFCC3A-DB2C-424C-B029-7FE99A87C641}"))
+    ...
+    """
+    _fields_ = [
+        ("Data1", wintypes.DWORD),
+        ("Data2", wintypes.WORD),
+        ("Data3", wintypes.WORD),
+        ("Data4", wintypes.BYTE * 8)
+    ]
+
+    def __init__(self, uuid_):
+        ctypes.Structure.__init__(self)
+        self.Data1, self.Data2, self.Data3, self.Data4[0], self.Data4[1], rest = uuid_.fields
+        for i in range(2, 8):
+            self.Data4[i] = rest >> (8 - i - 1)*8 & 0xff
+
+
+class UserHandle:
+    """
+    Class to provided User handles for the 'Default' and 'Current' Windows User
+
+    Returns
+    -------
+    C Pointer to user handle
+
+    Examples
+    --------
+    >>> UserHandle.common
+    ...
+    """
+    current = wintypes.HANDLE(0)
+    common = wintypes.HANDLE(-1)
```

Comment:
I'm not sure this should be a class... It's not really doing anything that a class should do.

Reply:
`UserHandle.common` doesn't appear to be used anywhere, except in a doc string example.

Reply:
We can remove it an just use the `wintypes.HANDLE(0)` directly. I left it as more of a future way if sunshine ever uses the default user path

Reply:
Removed the class and just used `wintypes.HANDLE(0)` directly as stated above